crates/staged-sync/src/config.rs

```diff
@@ -113,7 +113,7 @@ impl Default for BodiesConfig {
         Self {
             downloader_request_limit: 200,
             downloader_stream_batch_size: 10000,
-            downloader_max_buffered_responses: 30000,
+            downloader_max_buffered_responses: 1000,
```

Comment:
We should specify in the doc comment above that a response contains multiple blocks